frontend/trpc/routes/paymentManagement.ts

```
@@ -0,0 +1,280 @@
+
+        if (!response.ok) {
+          throw new TRPCError({
+            code: "INTERNAL_SERVER_ERROR",
+            message: `Failed to fetch account balances: ${response.statusText}`,
+          });
+        }
+
+        const balances = await response.json();
+        return balances;
+      } catch (error) {
+        console.error("Error fetching account balances:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)